features/autoconsent.json

```
@@ -811,7 +815,8 @@
             "bafin.de",
             "levi.pt",
             "premiumegeszsegpenztar.hu",
-            "mapquest.com"
+            "mapquest.com",
+            "ib3.org"
```

This is not caused by Easylist rules, filterlist exception is not necessary.
```suggestion
            "mapquest.com"
```